package.json

```diff
@@ -27,7 +27,7 @@
     "examples"
   ],
   "scripts": {
-    "lint": "eslint --fix './src/**/*.ts'",
```

Comment:
did you remove it by mistake?

Reply:
nope. it shouldn't out fix, but raise errors instead